README.md

```diff
@@ -1,24 +1,26 @@
 # MinIO Go Client SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Sourcegraph](https://sourcegraph.com/github.com/minio/minio-go/-/badge.svg)](https://sourcegraph.com/github.com/minio/minio-go?badge) [![Apache V2 License](https://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/minio/minio-go/blob/master/LICENSE)
 
-The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage.
+The MinIO Go Client SDK provides APIs to access any Amazon S3 compatible object storage.
```

Comment:
```suggestion
The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage.
```

Simple is on purpose in competing SDK AWS SDK Go is horrendous.

Reply:
What about `straightforward` rather than `simple`.

`simple` often carries a connotation that it should be easy, even if you don't have the requisite back knowledge. We try to avoid it in technical writing because it can often overpromise with how people read it.

Reply:
What @djwfyi said about _simple_ is exactly why I removed it. Straightforward is a good alternative. 👍🏻